templates/taxi/car_detail.html

```diff
@@ -3,13 +3,15 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
+      {% if user in car.drivers.all %}
```

Comment:
The condition `{% if user in car.drivers.all %}` checks if the current user is in the list of drivers for the car. Ensure that `car.drivers.all` is a callable method, so it should be `car.drivers.all()` to properly evaluate the queryset.